lib/array_intersection.rb

```diff
@@ -1,3 +1,22 @@
 def intersection(list1, list2)
```

Comment:
This works, although you don't need to worry about duplicates in the list as the prompt says every element in each list is unique.